admin.go

```diff
@@ -479,7 +503,19 @@ func (ca *clusterAdmin) AlterConfig(resourceType ConfigResourceType, name string
 		ValidateOnly: validateOnly,
 	}
 
-	b, err := ca.Controller()
+	var (
+		b   *Broker
+		err error
+	)
+
+	// AlterConfig of broker must be sent to the broker in question
+	if resourceType == BrokerResource {
+		id, _ := strconv.Atoi(name)
+		b, err = ca.findBroker(int32(id))
+		_ = b.Open(ca.client.Config())
+	} else {
+		b, err = ca.Controller()
```

Comment:
Similarly to `DescribeConfig`, we don't need to send the request to the controller here, any broker can be used if the resource is not a specific broker and not a broker logger.